app/assets/stylesheets/tpi/_typography.scss

```diff
@@ -14,18 +14,22 @@
 
 h1 {
   @extend .is-size-1;
+  font-weight: bold;
```

Comment:
I've added almost the same thing but don't use weight, use the bold font variation.

Reply:
Ok, I checked your PR and see what you mean